infra-as-code/bicep/machinelearning.bicep

```diff
@@ -125,10 +125,9 @@ resource blobStorageContributorForUserRoleAssignment 'Microsoft.Authorization/ro
   properties: {
     roleDefinitionId: storageBlobDataContributorRole.id
     principalType: 'User'
-    principalId: yourPrincipalId  // Production readiness change: Users shouldn't be using the Prompt flow developer portal in production, so this role
-                                  // assignment would only be needed in pre-production environments. In pre-production, use conditions on this assignment
-                                  // to restrict access to just the blob containers used by the project.
-
+    principalId: yourPrincipalId // Production readiness change: Users shouldn't be using the Prompt flow developer portal in production, so this role
+    // assignment would only be needed in pre-production environments. In pre-production, use conditions on this assignment
+    // to restrict access to just the blob containers used by the project.

```

Comment:
Please align with line 128. We align comments vertically when there are multiple in the same indent level in the same scope.